Tools/peg_generator/pegen/__main__.py

```diff
@@ -107,7 +107,13 @@ def generate_python_code(
     help="Suppress code emission for rule actions",
 )
 
-python_parser = subparsers.add_parser("python", help="Generate Python code")
+python_parser = subparsers.add_parser(
+    "python",
+    help=(
+        "Generate Python code, needs grammar definition with Python actions, "
```

Comment:
I would say `needs Python-specific grammar file (different from cpython/Grammar/python.gram which is for parser in C), `